docs/docker-stack/docker-operator-with-localexecutor-in-docker-compose.md

```diff
@@ -0,0 +1,87 @@
+# Using the DockerOperator with the LocalExecutor in Docker Compose
+
+This guide will allow you to run the DockerOperator using the LocalExecutor with
+Apache Airflow deployed on Docker Compose. The guide is split into four consecutive steps:
+
+* 1. Preparing the docker-compose.yaml
+* 2. Adding a new services in the docker-compose.yaml
+* 3. Creating a DAG that connects to the Docker API using this proxy
+* 4. Testing the DockerOperator
+
+To deploy Airflow on Docker  with Docker Compose you should fetch the docker-compose.yaml:
+
+```
+curl -LfO 'https://airflow.apache.org/docs/apache-airflow/2.1.2/docker-compose.yaml'
+```
+
+For more information about running Airflow in Docker refer to the following
+[documentation](https://airflow.apache.org/docs/apache-airflow/stable/start/docker.html).
+
+## 1. Preparing the docker-compose.yaml
+
+First you need to adapt the standard docker-compose.yaml file to work with
+the LocalExecutor and DockerOperator:
+
+1. line 50: replace `CeleryExecutor` with `LocalExecutor`
```

Comment:
I wonder if it is worth dividing it into two guides. One that will describe how to use LocalExecutor and the other how to use Docker Operator.